compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/platform/BackingTextArea.kt

```diff
@@ -176,20 +201,15 @@ internal class BackingTextArea(
     }
 
     fun updateState(textFieldValue: TextFieldValue) {
-        textArea.value = textFieldValue.text
-        textArea.setSelectionRange(textFieldValue.selection.start, textFieldValue.selection.end)
+//        textArea.value = textFieldValue.text
+//        textArea.setSelectionRange(textFieldValue.selection.start, textFieldValue.selection.end)
```

Comment:
Had you wanted to delete this?

Reply:
Yep, this is code that will be fixed separately while addressing 
https://youtrack.jetbrains.com/issue/CMP-1436

Reply:
It would be better to have a note about it right before the code